.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Mnóstwo jakiegoś badziewia się dostało, ale sama nie wiem, jak to zatrzymać tak na szybko - trzbaby plik .gitignore dać.